packages/react-app/src/views/GuestRoom.jsx

```diff
@@ -41,6 +42,20 @@ export default function GuestRoom({
 
   const subs = useRef([]);
 
+  useEffect(() => {
+    // moving current user to the top of the list
+    if (addresses && addresses.length > 0) {
+      const newAddresses = [...addresses];
+      newAddresses.forEach((add, index) => {
+        if (add === address) {
```

Comment:
Comparing it with lowercase might make it work:
(add.toLowerCase() === address.toLowerCase())

Reply:
Thanks for suggesting. Added that